src/components/toggle.jsx

```diff
@@ -7,33 +7,36 @@ class ToggleContainerViews extends React.Component {
   constructor(props) {
     super();
 
-    const view = props.defaultView;
+    const view = props.defaults.view;
```

Comment:
This will be in the app state passed down through props. 

Instead of setting local state from the props you could just use the this.props.store.state to decide the view